task_1.py

```diff
@@ -0,0 +1,10 @@
+numbers = [2, -93, -2, 8, None, -44, -1, -85, -14, 90, -22, -90, -100, -8, 38, -92, -45, 67, 53, 25]
+
+
+# TODO заменить значение пропущенного элемента средним арифметическим
+incorrect_none = numbers[4]
+correct_none = 0
+numbers[4] = correct_none
+numbersum = (sum(numbers) / len(numbers))
+numbers[4] = numbersum
```

Comment:
Тут тоже можно применить небольшое сокращение строк кода и сразу после замены `None` на `0`, произвести постановку искомого значения
```suggestion
numbers[4] = (sum(numbers) / len(numbers))
```